accelerometer.h

```diff
@@ -19,14 +19,25 @@
 
 #include <anjay/anjay.h>
 
-#ifdef TARGET_DISCO_L496AG
+#if (SENSORS_IKS01A2 == 1)
 
 int accelerometer_object_install(anjay_t *anjay);
 
 void accelerometer_object_uninstall(anjay_t *anjay);
 
 void accelerometer_object_update(anjay_t *anjay);
 
-#endif // TARGET_DISCO_L496AG
+#else // Define dummy functions, compiler optimizes out
+
+int accelerometer_object_install(anjay_t *anjay) {
+    return 0;
+}
+void accelerometer_object_uninstall(anjay_t *anjay) {
+}
+
+void accelerometer_object_update(anjay_t *anjay) {
+}
```

Comment:
These (and similar declarations in other headers) should either be moved to the `.cpp` files, or declared as `static inline`, or put into an unnamed namespace. Otherwise, link errors may occur if this header is included from multiple translation units.

Reply:
Static inline seems to be how to do it Linux. I'll do that.

Reply:
Done.